server.go

```diff
@@ -537,25 +544,27 @@ func getLeaderboardData(start time.Time) ([]LeaderboardEntry, error) {
 		log.Printf("error getting leaderboard data from database - %v", err)
 		return leaderboardData, errors.New(ErrDB)
 	}
+	defer rows.Close()
```

Comment:
whenever you access db rows, you have to defer the close so we release the connection back to the pool